constructor/main.py

```diff
@@ -350,31 +335,25 @@ def _build_message(self):
 
             Available selectors
             -------------------
-
             Constructor can use the same Selector enhancement of the YAML format
             used in conda-build ('# [selector]'). Available keywords are:
 
             {available_selectors}
             """
         )
-        available_keys_list = []
-        for key, required, key_types, help_msg, plural in generate_key_info_list():
-            available_keys_list.append(
-                "\n".join(
-                    [
-                        key,
-                        "·" * len(key),
-                        indent(f"Required: {required}, type{plural}: {key_types}", "    "),
```

Comment:
This part I have omitted from the autogenerated docs. I can't simplify the type annotation enough for readability and I hope users are checking the schema (or its errors) anyway.

Reply:
I think we'll be ok. This is mostly for local users that don't want to open their browser to read the docs and just want to check the available keys, but with the JSON Schema auto completion we don't need this. We can deprecate it. WDYT?

Reply:
I'm fine with deprecating it or using the minimal version of this PR. It was just a thought and it's not a blocker.